src/webui/router.js

```diff
@@ -12,6 +12,7 @@ import { AppContextConsumer } from './app';
 import { asyncComponent } from './utils/asyncComponent';
 import history from './history';
 import Header from './components/Header';
+import NotFound from './components/NotFound';
```

Comment:
I'd recommend load import dynamically for Webpack code splitting. 
```
const NotFound = asyncComponent(() => import('./components/NotFound'));
```

Reply:
I will change ...thank you for the Tip :-)